pkg/machine/qemu/options_linux_amd64.go

```diff
@@ -0,0 +1,18 @@
+package qemu
+
+var (
+	QemuCommand = "qemu-kvm"
```

Comment:
This Fedora-specific command is just a symlink: /usr/bin/qemu-kvm -> qemu-system-x86_64

Better to use the real command (like on darwin), and then add -enable-kvm (or: -accel kvm)

Reply:
And then same for linux_arm64, but run qemu-system-aarch64 there ?

Reply:
thank you, its on my list for sure

Reply:
We used to have a test for `/dev/kvm`, just in case you were running on a different system or so.

But just wanted to avoid: `Error: exec: "qemu-kvm": executable file not found in $PATH`

Reply:
where "different system" mostly meant windows, but could theoretically also be freebsd